### ISSUE TITLE
Consistency in limit notation

### ISSUE THREAD

**Issue by AbbyANoble** [TeamBasedInquiryLearning/calculus#138]
In [Activity 1.2.4](https://teambasedinquirylearning.github.io/calculus/2024e/LT2.html#activity-limits-numerically3), sometimes the "x-> something" notation is under lim and sometimes it is beside it. I think we should have consistency unless there is a specific case where one makes more sense. Personally, I would prefer the "x-> something" to be under the lim. We've defined it that way in LT1.

I'm also seeing the inline one in:

-  [Activity 1.2.1](https://teambasedinquirylearning.github.io/calculus/2024e/LT2.html#activity-limits-numerically1)  
- [Theorem 1.1.11](https://teambasedinquirylearning.github.io/calculus/2024e/LT1.html#squeeze-thm)
- [Activity 1.1.12](https://teambasedinquirylearning.github.io/calculus/2024e/LT1.html#squeeze-thm-ex) 
- [Activity 1.2.5](https://teambasedinquirylearning.github.io/calculus/2024e/LT2.html#activity-limits-numerically4) -- part (a) does under and part (b) does inline
- [Activity 1.4.4](https://teambasedinquirylearning.github.io/calculus/2024e/LT4.html#activity-continuous-graph1)
- [Definition 1.4.5](https://teambasedinquirylearning.github.io/calculus/2024e/LT4.html#definition-continuity) 
- [Activity 1.4.7](https://teambasedinquirylearning.github.io/calculus/2024e/LT4.html#activity-continuous-graph2) 
- [Remark 1.4.11](https://teambasedinquirylearning.github.io/calculus/2024e/LT4.html#LT4-3-12)

**Comment by siwelwerd** [comment]
The fix is to make sure to drop a `\displaystyle` into the `<m>` or `<me>`

Here's the source for the first choice in [1.2.4](https://teambasedinquirylearning.github.io/calculus/2024e/LT2.html#activity-limits-numerically3):
https://github.com/TeamBasedInquiryLearning/calculus/blob/733d0f1d73240406dbf6c31426bfd4659c463ff7/source/01-LT/02.ptx#L221

Note how there is a `\displaystyle` in the first `<m>`, but not the second.

I agree that the `\displaystyle` approach is preferable and that consistency is needed.

**Comment by StevenClontz** [comment]
FWIW I believe `\displaystyle` is only effective in `<m/>` tags, as `<me/>` already renders math in displaymath mode.

**Comment by AbbyANoble** [comment]
Assuming there's a find and replace way to do this rather than combing through every lesson? 🤞 My list above was not complete. Just what I found with some quick glancing.

**Comment by siwelwerd** [comment]
Maybe a regex looking for a `<m>` followed by a `\lim` without a `\displaystyle` in between?

**Comment by siwelwerd** [comment]
```
(base) drew@Drews-MBP calculus % grep -n '\\lim' source/*/*.ptx | grep -v "displaystyle" | grep -v '<me>' | grep -v '<mrow>'
source/01-LT/01.ptx:114:            \lim_{x \to a} f(x) = L
source/01-LT/01.ptx:156:            \lim_{x \to a^-} f(x) = L_1
source/01-LT/01.ptx:162:            \lim_{x \to a^+} f(x) = L_2
source/01-LT/01.ptx:272:                  <li> <p> the limit as <m>x</m> approaches <m>a</m> of <m> f(x) </m> is equal to the same value <m>L</m> as the limit of <m>h(x)</m>, so <m>\lim_{x\to a} f(x) = L =\lim_{x\to a} h(x) .</m></p></li>
source/01-LT/01.ptx:274:<p>Then, the limit of <m>g(x)</m> as <m>x \to a</m> is also <m> L</m>, so <m>\lim_{x\to a} g(x) = L </m>. </p></statement> </theorem>
source/01-LT/01.ptx:308:                <p> Using trigonometry, one can show algebraically that <m>\cos(x) \leq \frac{\sin(x)}{x} \leq 1 </m> for <m>x</m> values close to zero. Moreover, <m>\lim_{x \to 0} \cos(x) = \cos(0)=1</m> (we say that cosine is a continuous function). Use these facts and the Squeeze Theorem, to find the limit <m>\lim_{x\to 0} \frac{\sin(x)}{x} </m>.</p>
source/01-LT/02.ptx:73:        Based on the values of <xref ref="table-limit-numerical1"/>, what is the best approximation for <m>\lim_{x\to 7} f(x)</m>?
source/01-LT/02.ptx:246:        <p> Use the values that you chose in part (a) to calculate an approximation for <m>\lim_{x \to 2}f(x)</m>. </p>
source/01-LT/03.ptx:155:                \lim_{x \to c} f(x) = L \, \text{and} \, \lim_{x \to c} g(x) = K,
source/01-LT/03.ptx:188:                \lim_{x \to c} f(x) = L, \,  \, \, \, \lim_{x \to L} g(x) = K,  \text{and} \, g(L)=K.
source/01-LT/03.ptx:327:        <statement><p>Consider taking the limit of a rational function <m>\frac{p(x)}{q(x)}</m> as <m>x \to c</m>. If <m>q(c)=0</m>, is it possible for <m>\lim_{x \to c}\frac{p(x)}{q(x)} </m> to equal a number? </p> 
source/01-LT/03.ptx:331:            <li>No, because if you graph <m>g(x)=\frac{x^2+1}{x-1}</m>, the value <m>g(1)</m> is not defined and the graph shows that the limit of <m>\lim_{x \to c}g(x)</m> does not exist.</li>
source/01-LT/04.ptx:70:              For each of the values <m>a = -3</m>, <m>-2</m>, <m>-1</m>, <m>0</m>, <m>1</m>, <m>2</m>, <m>3</m>, determine whether the limit <m>\lim_{x \to a} f(x)</m> exists. If the limit does not exist, be ready to explain why not.
source/01-LT/04.ptx:80:              For each such <m>a</m> value, is <m>f(a)</m> equal to <m>\lim_{x \to a} f(x)</m>?
source/01-LT/04.ptx:112:                <m>\lim_{x \to a} f(x) = f(a)</m>.
source/01-LT/04.ptx:128:              <m>\lim_{x \to -3^+} h(x)</m>
source/01-LT/04.ptx:133:              <m>\lim_{x \to -3^-} h(x)</m>
source/01-LT/04.ptx:138:              <m>\lim_{x \to -3} h(x)</m>
source/01-LT/04.ptx:164:              For which values of <m>a</m> do we have <m>\lim_{x \to a^-} f(x) \ne \lim_{x \to a^+} f(x)</m>?
source/01-LT/04.ptx:174:              For which values of <m>a</m> does <m>f</m> have a limit at <m>a</m>, yet <m>f(a) \ne \lim_{x \to a} f(x)</m>?
source/01-LT/04.ptx:288:          When <m>\lim_{x \to a} f(x)</m> exists but is not equal to <m>f(a)</m>, we say that <m>f</m> has a <term>removable discontinuity</term><idx>removable discontinuity</idx> at <m>x = a</m>. This is because if <m>f(a)</m> were redefined to be equal to <m>\lim_{x \to a} f(x)</m>, the redefined function would be continuous at <m>x = a</m>, thus <q>removing</q> the discontinuity.
source/01-LT/05.ptx:41:    \lim_{x \to + \infty} x^3 = + \infty \, ,  \, \lim_{x \to - \infty} x^3 = - \infty. </me>
source/01-LT/05.ptx:71:    \lim_{x\to +\infty} f(x) = b
source/01-LT/05.ptx:75:    \lim_{x\to -\infty} f(x) = b
source/01-LT/05.ptx:299:\lim_{x\to-\infty } -\frac{6 \, {x^4} + 7 \, {x^3} - 7}{6 \, x - {x^4} + 9}
source/01-LT/05.ptx:301:\lim_{x\to+\infty } -\frac{6 \, {x^4} + 7 \, {x^3} - 7}{6 \, x - {x^4} + 9}
source/01-LT/05.ptx:304:\lim_{x\to-\infty } -\frac{7 \, {x^4} - 5 \, {x^3} + 8}{3 \, {\left(2 \, {x^5} + 3 \, {x^2} - 3\right)}}
source/01-LT/05.ptx:306:\lim_{x\to+\infty } -\frac{7 \, {x^4} - 5 \, {x^3} + 8}{3 \, {\left(2 \, {x^5} + 3 \, {x^2} - 3\right)}}
source/01-LT/05.ptx:309:\lim_{x\to-\infty } \frac{3 \, {x^6} + {x^3} - 8}{7 \, x - 6 \, {x^5} + 7}
source/01-LT/05.ptx:311:\lim_{x\to+\infty } \frac{3 \, {x^6} + {x^3} - 8}{7 \, x - 6 \, {x^5} + 7}
source/01-LT/05.ptx:317:\lim_{x\to-\infty } -\frac{6 \, {x^4} + 7 \, {x^3} - 7}{6 \, x - {x^4} + 9} = 6
source/01-LT/05.ptx:319:\lim_{x\to+\infty } -\frac{6 \, {x^4} + 7 \, {x^3} - 7}{6 \, x - {x^4} + 9} = 6
source/01-LT/05.ptx:322:\lim_{x\to-\infty } -\frac{7 \, {x^4} - 5 \, {x^3} + 8}{3 \, {\left(2 \, {x^5} + 3 \, {x^2} - 3\right)}} = 0
source/01-LT/05.ptx:324:\lim_{x\to+\infty } -\frac{7 \, {x^4} - 5 \, {x^3} + 8}{3 \, {\left(2 \, {x^5} + 3 \, {x^2} - 3\right)}} = 0
source/01-LT/05.ptx:327:\lim_{x\to-\infty } \frac{3 \, {x^6} + {x^3} - 8}{7 \, x - 6 \, {x^5} + 7} = +\infty
source/01-LT/05.ptx:329:\lim_{x\to+\infty } \frac{3 \, {x^6} + {x^3} - 8}{7 \, x - 6 \, {x^5} + 7} = -\infty
source/01-LT/05.ptx:458:    <p> An exponential function <m>P(t) = a \, b^t</m> exhibiting exponential decay will have the long term behavior <m>P(t) \to 0</m> as <m> t \to \infty</m>. If we shift the graph up by <m>c</m> units, we obtain the new function <m>Q(t) = a \, b^t + c</m>, with the long term behavior <m>\lim_{t \to \infty} Q(t) = c</m>. A cooling object can be represented by the exponential decay model <m>Q(t) = a \, b^t + c</m>. </p>
source/01-LT/06.ptx:58:    \lim_{x\to a} f(x) = + \infty
source/01-LT/06.ptx:62:    \lim_{x\to a} f(x) = - \infty
source/02-DF/01.ptx:118:     v(2) = \lim_{h\to 0} \frac{f(2+h)-f(2)}{h}</me>    
source/02-DF/01.ptx:245:         <li><p> The limit <m>\lim_{h \to 0} \frac{f(a+h)-f(a)}{h} </m> </p></li>                    
source/02-DF/01.ptx:428:<m>\lim_{h\to 0} \frac{f( 2 +h)-f( 2 )}{h}&lt; 0</m>
source/03-AD/09.ptx:158:\lim_{x\to \infty } \frac{ -8 \, x + 3 \, e^{x} }{ 7 \, x - 3 \, e^{x} }
source/03-AD/09.ptx:165:\lim_{x\to 0 } \frac{ 6 \, \cos\left(8 \, x\right) }{ 4 \, x - 7 }
source/03-AD/09.ptx:172:\lim_{x\to 0 } \frac{ -9 \, \cos\left(3 \, x\right) + 9 }{ -3 \, x }
source/03-AD/09.ptx:179:\lim_{x\to 4 } \frac{ x^{2} - x - 12 }{ x^{2} - 13 \, x + 36 }
source/03-AD/09.ptx:189:\lim_{x\to \infty } \frac{ -8 \, x + 3 \, e^{x} }{ 7 \, x - 3 \, e^{x} } = -1
source/03-AD/09.ptx:201:\lim_{x\to 0 } \frac{ -9 \, \cos\left(3 \, x\right) + 9 }{ -3 \, x } = 0
source/03-AD/09.ptx:208:\lim_{x\to 4 } \frac{ x^{2} - x - 12 }{ x^{2} - 13 \, x + 36 } = -\frac{7}{5}
source/05-TI/08.ptx:120:            <li><m>\lim_{b\to\infty}\int_1^b\frac{1}{x^2}dx</m></li>
source/05-TI/08.ptx:121:            <li><m>\lim_{b\to\infty}\left[-\frac{1}{x}\right]_1^b</m></li>
source/05-TI/08.ptx:122:            <li><m>\lim_{b\to\infty}\left[-\frac{1}{b}+1\right]</m></li>
source/05-TI/08.ptx:157:            <li><m>\lim_{a\to0^+}\int_a^1\frac{1}{\sqrt x}dx</m></li>
source/05-TI/08.ptx:158:            <li><m>\lim_{a\to0^+}\left[2\sqrt x\right]_a^1</m></li>
source/05-TI/08.ptx:159:            <li><m>\lim_{a\to0^+}\left[2-2\sqrt a\right]</m></li>
source/06-AI/04.ptx:397:                    \lim_{\Delta x\to 0} \frac{f(x_i)+f(x_i+\Delta x)}{2}=f(x_i).
source/08-SQ/06.ptx:139:          \lim_{n \to \infty} \frac{b_n}{a_n} = c
source/08-SQ/06.ptx:193:          \lim_{n \to \infty} \frac{b_n}{a_n}
```

**Comment by siwelwerd** [comment]
That is an overcount, some of those are in a `<me>` that started on a previous line. But maybe combing through that list and fixing them all would be a good start.

**Comment by AbbyANoble** [comment]
😳 That sounds like a job for future Abby. Or current/future somebody else. Haha. :)

**Comment by siwelwerd** [comment]
I bet it would go faster than you think, but you'll notice I am also not volunteering to do it at this moment 🙃

**Comment by StevenClontz** [comment]
Quick and dirty PR proposed at #139 which may close this.

**Comment by StevenClontz** [comment]
Er, #139 bugged out, but #140 is what we want I think.

**Comment by siwelwerd** [comment]
Went ahead and merged #140, which I think closes this.
